contrib/debian/systemd/ebusd.service

```diff
@@ -7,7 +7,7 @@ ConditionPathExists=/var/log
 Type=forking
 Restart=always
 RestartSec=30
-PIDFile=/var/run/ebusd.pid
+PIDFile=/run/ebusd.pid
```

Comment:
according to https://www.gnu.org/prep/standards/html_node/Directory-Variables.html it should be /var/run and not /run

Reply:
according to systemd document:
https://www.freedesktop.org/software/systemd/man/systemd.exec.html#RuntimeDirectory=
RuntimeDirectory= /run/

Reply:
then it would have to be done differently for initd/systemd. plus you'd also have to tell ebusd about the changed location

Reply:
this you know better than, I just wanted to point it out that ev. path change is required.
thanks

Reply:
even [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) of recent cmake still has LOCALSTATEDIR/run in it with LOCALSTATEDIR usually being /var, so I'll keep it until that changes